pkgs/top-level/all-packages.nix

```diff
@@ -29000,6 +29000,7 @@ with pkgs;
 
   tdesktop = libsForQt5.callPackage ../applications/networking/instant-messengers/telegram/tdesktop {
     inherit (xorg) libpthreadstubs libXdmcp;
```

Comment:
```suggestion
```

`xorg` packages are automatically injected by `callPackage` so this line is redundant.